ci/test_python_other.sh

```diff
@@ -54,5 +55,18 @@ rapids-logger "pytest custreamz"
   --cov-report=xml:"${RAPIDS_COVERAGE_DIR}/custreamz-coverage.xml" \
   --cov-report=term
 
+# Note that cudf-polars uses rmm.mr.CudaAsyncMemoryResource() which allocates
```

Comment:
I saw you made a related comment on this [issue](https://github.com/rapidsai/cudf/issues/16723#issuecomment-2417611328), should we include a link to that issue in this comment?

Reply:
Done in 8349420d5437dbd7c7d47930423ba6a8d50d9ddd.